code/mod_table/mod_table.cpp

```diff
@@ -1667,6 +1675,7 @@ void mod_table_set_version_flags()
 		Shockwaves_damage_all_obj_types_once = true;
 		Framerate_independent_turning = true;					// this is already true, but let's re-emphasize it
 		Use_host_orientation_for_set_camera_facing = true;		// this is essentially a bugfix
+		Use_model_eyepoint_for_set_camera_facing = true;
```

Comment:
It's good to add this flag as a bugfix, as you did here, but it's in the wrong section.  Any new flags added should be associated with the upcoming stable version, so this should be added in the 25.0.0 section.  (Currently, that section doesn't exist, so you'll need to create it.)